booking-engine-api/operations/services.md

```diff
@@ -0,0 +1,354 @@
+              "NetValue": 38.83,
+              "Breakdown": {
+                "Items": [
+                  {
+                    "TaxRateCode": null,
+                    "NetValue": 9.06,
+                    "TaxValue": 0.0
+                  },
+                  {
+                    "TaxRateCode": "CZ-L",
+                    "NetValue": 28.75,
+                    "TaxValue": 2.88
+                  },
+                  {
+                    "TaxRateCode": "CZ-Z",
+                    "NetValue": 1.02,
+                    "TaxValue": 0.00
+                  }
+                ]
+              }
+            }
+          },
+          "MaxPrice": {
+            "TotalAmount": {
+              "Currency": "EUR",
+              "GrossValue": 20020.15,
+              "NetValue": 18201.97,
+              "Breakdown": {
+                "Items": [
+                  {
+                    "TaxRateCode": null,
+                    "NetValue": 18.11,
+                    "TaxValue": 0.0
+                  },
+                  {
+                    "TaxRateCode": "CZ-L",
+                    "NetValue": 18181.82,
+                    "TaxValue": 1818.18
+                  },
+                  {
+                    "TaxRateCode": "CZ-Z",
+                    "NetValue": 2.04,
+                    "TaxValue": 0.00
+                  }
+                ]
+              }
+            },
+            "AverageAmountPerTimeUnit": {
+              "Currency": "EUR",
+              "GrossValue": 10010.08,
+              "NetValue": 9100.99,
+              "Breakdown": {
+                "Items": [
+                  {
+                    "TaxRateCode": null,
+                    "NetValue": 9.06,
+                    "TaxValue": 0.0
+                  },
+                  {
+                    "TaxRateCode": "CZ-L",
+                    "NetValue": 9090.91,
+                    "TaxValue": 909.09
+                  },
+                  {
+                    "TaxRateCode": "CZ-Z",
+                    "NetValue": 1.02,
+                    "TaxValue": 0.00
+                  }
+                ]
+              }
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RateGroups` | array of [Rate group](hotels.md#rate-group) | required | Information about all available rate groups. |
+| `Rates` | array of [Rate](hotels.md#rate) | required | Information about all available rates. |
+| `CategoryPrices` | array of [Category price](#category-price) | required | Prices for all specified categories. |
+
+#### Category price 
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | required | Unique identifier of category. |
+| `OccupancyPrices` | array of objects with arrays of [Occupancy](#occupancy) | required | List of occupancies for age categories against which rate group prices are supplied. |
```

Comment:
It is due to multiple `Occupancies` in `OccupancyPrices` can have the same `RateGroupPrices` - it would unnecessarily duplicate the data with your grouping approach